src/Microsoft.VisualBasic/tests/VB/SpecialDirectoriesTests.vb

```diff
@@ -14,6 +14,15 @@ Namespace Microsoft.VisualBasic.Tests.VB
             IO.Path.AltDirectorySeparatorChar
             }
 
+        Private Shared Sub CheckSpecialFolder(folder As SpecialFolder, getSpecialDirectory As Func(Of String))
+            If Environment.GetFolderPath(folder) = "" Then
+                Assert.Throws(Of IO.DirectoryNotFoundException)(Function() getSpecialDirectory)
```

Comment:
Can pass `getSpecialDirectory` as the argument to `Assert.Throws()` rather than `Function() getSpecialDirectory`.

Reply:
It seems like it doesn't throw unless I .Invoke it. I am testing change, will let you know how it goes.